flake.nix

```diff
@@ -0,0 +1,38 @@
+{
+  description = "An optional nix-based development setup";
+
+  inputs.flake-utils.url = "github:numtide/flake-utils";
+
+  outputs = { self, nixpkgs, flake-utils }:
+    flake-utils.lib.eachDefaultSystem (system:
+      let pkgs = nixpkgs.legacyPackages.${system}; in
+      {
+        apps = {
+          buildVscodeExtension = pkgs.writeShellScriptBin "buildVscodeExtension" ''
+              PATH=${pkgs.nodejs}/bin:$PATH
+
+              pushd ./packages/vscode
+              npm install
+              npm run build
+              npm run package
+              popd
+          '';
+          code = pkgs.writeShellScriptBin "code" ''
+            USER_DIR=`mktemp -d`
+            DATA_DIR=`mktemp -d`
+            CODE="${pkgs.vscodium}/bin/codium --user-data-dir $USER_DIR"
+            EXTENSION_FILE=`find ./packages/vscode -name 'prisma-insider*.vsix' -print0`
+
+            if [[ $EXTENSION_FILE == "" ]]; then
+              echo "Could not find extension file in packages/vscode"
+              exit 1
+            fi
+
+            mkdir $USER_DIR $DATA_DIR
+            $CODE --install-extension $EXTENSION_FILE
+            $CODE $DATA_DIR
```

Comment:
This could maybe open up a prisma file in that directory?

Reply:
yep it could, good idea

Reply:
done